test/hotspot/jtreg/serviceability/sa/sadebugd/ClhsdbTestConnectArgument.java

```diff
@@ -75,7 +75,10 @@ public static void main(String[] args) throws Exception {
             System.out.println(out.getStdout());
             System.err.println(out.getStderr());
 
-            out.stderrShouldBeEmptyIgnoreDeprecatedWarnings();
+            out.shouldMatch("WARNING: --connect is deprecated");
```

Comment:
You might want to consider improving stderrShouldBeEmptyIgnoreDeprecatedWarnings(). It could, for instance, do a case insensitive search for "warning" and "deprecated" rather than the more specific search it is currently doing. Just a suggestion. Don't bother if you don't think it is worth it.

Reply:
I had looked at doing that but was worried about missing other real messages.
However looking again, to match "^WARNING: .* is deprecated.*" seems quite unambiguous!
Will update...